Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -1727,6 +1727,7 @@ namespace AzToolsFramework
 
                 AliasPath absoluteInstancePath = commonOwningInstance.GetAbsoluteInstanceAliasPath();
                 absoluteInstancePath.Append(newInstanceAlias);
+                absoluteInstancePath.Append("ContainerEntity");
```

Comment:
Instead of hardcoding the name here, consider using `ContainerEntityName`.

Reply:
In fact there is one already here:  ```PrefabDomUtils::ContainerEntityName```